### DISCUSSION TITLE
UDM SE, KPN, stb-nmc-400 error

### DISCUSSION THREAD

**Discussion by VentingBow5** [fabianishere/udm-iptv#135] [Q&A]
I installed new UDM SE last week, followed the guide and everything is working fine but KPN TV freezes after 10 min with error message stb-nmc-400 (De verbinding tussen de tv-ontvanger en je modem is waarschijnlijk verbroken), changing between channels solves it. However, 10 min later I have the same message.

What am I missing?

Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth8
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br0
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
25: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.242.130.175/22 brd 10.242.131.255 scope global iptv
       valid_lft forever preferred_lft forever
10.242.128.0/22 proto kernel scope link src 10.242.130.175
213.75.112.0/21 via 10.242.128.1 metric 225
=== Service Logs ===
Jun 11 08:49:41 UDM-SE igmpproxy[1497]: No interfaces found for source 192.168.1.69
Jun 11 08:49:41 UDM-SE igmpproxy[1497]: No interfaces found for source 192.168.1.69
Jun 11 08:49:41 UDM-SE igmpproxy[1497]: No interfaces found for source 192.168.1.69
Jun 11 08:49:41 UDM-SE igmpproxy[1497]: No interfaces found for source 192.168.1.69
Jun 11 08:49:41 UDM-SE igmpproxy[1497]: No interfaces found for source 192.168.1.69
Jun 11 08:49:41 UDM-SE igmpproxy[1497]: No interfaces found for source 192.168.1.69
Jun 11 08:49:42 UDM-SE igmpproxy[1497]: No interfaces found for source 192.168.1.69
Jun 11 08:49:42 UDM-SE igmpproxy[1497]: No interfaces found for source 192.168.1.69
Jun 11 08:49:42 UDM-SE igmpproxy[1497]: No interfaces found for source 192.168.1.69
Jun 11 08:49:42 UDM-SE igmpproxy[1497]: No interfaces found for source 192.168.1.69

**Comment by fabianishere** [comment]
What is the IP range of your LAN network?
```bash
ip address show dev br0
```

Also, could you try restarting the service:
```bash
systemctl restart udm-iptv
```

**Reply by VentingBow5** [reply]
IP range is 192.168.2.6 - 192.168.2.254 (Auto Scale Network enabled). 

I have restarted the service. Will monitor and let you know whether this solved the problem.

**Reply by fabianishere** [reply]
Any idea what the device with IP address `192.168.1.69`. It looks like it is sending IGMP messages on `br0`.

**Reply by VentingBow5** [reply]
I was wandering the same as I do not have any device in my network with IP addres 192.168.1.69. Will try to ping it.

**Reply by VentingBow5** [reply]
The problem persists. Any other logs I can share that will help you diagnosing the issue?

Regarding the IP address 192.168.1.69: I realised I initially setup the network with a different IP range (192.168.1.1 etc). I completely removed the udm-itv installation and re-installed it. 

Diagnostics now look like:

Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth8
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br0
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
27: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.242.130.243/22 brd 10.242.131.255 scope global iptv
       valid_lft forever preferred_lft forever
10.242.128.0/22 proto kernel scope link src 10.242.130.243
213.75.112.0/21 via 10.242.128.1 metric 227
=== Service Logs ===
Jun 12 14:22:18 UDM-SE systemd[1]: Started IPTV support for the UniFi Dream Machine.
Jun 12 14:22:18 UDM-SE udm-iptv[24693]: Device iptv already exists.. Deleting device
Jun 12 14:22:18 UDM-SE udm-iptv[24693]: Obtaining IP address for VLAN interface
Jun 12 14:22:18 UDM-SE udm-iptv[24693]: udhcpc (v1.22.1) started
Jun 12 14:22:18 UDM-SE udm-iptv[24693]: Sending discover...
Jun 12 14:22:18 UDM-SE udm-iptv[24693]: Sending select for 10.242.130.243...
Jun 12 14:22:19 UDM-SE udm-iptv[24693]: Lease of 10.242.130.243 obtained, lease time 3600
Jun 12 14:22:19 UDM-SE udm-iptv[24693]: NATing IPTV network ranges (if necessary)
Jun 12 14:22:19 UDM-SE udm-iptv[24693]: Setting up igmpproxy
Jun 12 14:22:19 UDM-SE udm-iptv[24693]: Starting igmpproxy
root@UDM-SE:~#

**Comment by VentingBow5** [comment]
Via journalctl -u udm-iptv I pulled the below log:

-- Reboot --
Jun 12 14:07:48 UDM-SE systemd[1]: Started IPTV support for the UniFi Dream Machine.
Jun 12 14:07:48 UDM-SE udm-iptv[15915]: Obtaining IP address for VLAN interface
Jun 12 14:07:48 UDM-SE udm-iptv[15915]: udhcpc (v1.22.1) started
Jun 12 14:07:48 UDM-SE udm-iptv[15915]: Sending discover...
Jun 12 14:07:51 UDM-SE udm-iptv[15915]: Sending discover...
Jun 12 14:07:51 UDM-SE udm-iptv[15915]: Sending select for 10.242.130.243...
Jun 12 14:07:52 UDM-SE udm-iptv[15915]: Lease of 10.242.130.243 obtained, lease time 3600
Jun 12 14:07:52 UDM-SE udm-iptv[15915]: NATing IPTV network ranges (if necessary)
Jun 12 14:07:52 UDM-SE udm-iptv[15915]: Setting up igmpproxy
Jun 12 14:07:52 UDM-SE udm-iptv[15915]: Starting igmpproxy
Jun 12 14:22:18 UDM-SE systemd[1]: Stopping IPTV support for the UniFi Dream Machine...
Jun 12 14:22:18 UDM-SE igmpproxy[15915]: select() failure; Errno(4): Interrupted system call
Jun 12 14:22:18 UDM-SE systemd[1]: udm-iptv.service: Succeeded.
Jun 12 14:22:18 UDM-SE systemd[1]: Stopped IPTV support for the UniFi Dream Machine.
Jun 12 14:22:18 UDM-SE systemd[1]: udm-iptv.service: Consumed 204ms CPU time.
Jun 12 14:22:18 UDM-SE systemd[1]: Started IPTV support for the UniFi Dream Machine.
Jun 12 14:22:18 UDM-SE udm-iptv[24693]: Device iptv already exists.. Deleting device
Jun 12 14:22:18 UDM-SE udm-iptv[24693]: Obtaining IP address for VLAN interface
Jun 12 14:22:18 UDM-SE udm-iptv[24693]: udhcpc (v1.22.1) started
Jun 12 14:22:18 UDM-SE udm-iptv[24693]: Sending discover...
Jun 12 14:22:18 UDM-SE udm-iptv[24693]: Sending select for 10.242.130.243...
Jun 12 14:22:19 UDM-SE udm-iptv[24693]: Lease of 10.242.130.243 obtained, lease time 3600
Jun 12 14:22:19 UDM-SE udm-iptv[24693]: NATing IPTV network ranges (if necessary)
Jun 12 14:22:19 UDM-SE udm-iptv[24693]: Setting up igmpproxy
Jun 12 14:22:19 UDM-SE udm-iptv[24693]: Starting igmpproxy
